recipes/vtku3dexporter/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: vtku3dexporter
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/ClinicalGraphics/u3d.git
+  git_tag: {{ version }}
+
+build:
+  features:
+  - vc9  # [win and py27]
+  - vc10  # [win and py34]
+  - vc14  # [win and py35]
+  number: 0
+  skip: true  # [osx or win32 or linux32]
```

Comment:
For OSX you could always just try enabling the build and seeing if it passes on Travis. Alternatively I can have a look at some point (when I have my laptop in front of me). And with regards to Win32 - yes quite a lot of people unfortunately as it's still very common in the laptop market!